packages/engine/tests/units/state/immutability/src/behaviors/changing.js

```diff
@@ -0,0 +1,4 @@
+const behavior = (state, context) => {
+  state.myObject.value = 10;
```

Comment:
*nit*: We could set this in the `init.js` file (`init.py` file for the Python version` and set the value to `nullable: false`.